lib/praxis/restful_doc_generator.rb

```diff
@@ -285,7 +290,8 @@ def write_index( versioned_types )
 
         reportable_types.each do |type|
           index[version] ||= Hash.new
-          display_name = type.name.split("::").last
+          next unless type.respond_to?(:display_name)
```

Comment:
I think we need to skip it, otherwise if we try to guess we'll likely get conflicts again, no?